src/vs/platform/quickinput/browser/quickInputBox.ts

```diff
@@ -29,7 +29,7 @@ export class QuickInputBox extends Disposable {
 		this.container = dom.append(this.parent, $('.quick-input-box'));
 		this.findInput = this._register(new FindInput(this.container, undefined, { label: '', inputBoxStyles, toggleStyles }));
 		const input = this.findInput.inputBox.inputElement;
-		input.role = 'combobox';
+		input.role = 'input';
```

Comment:
Are you sure this is right? I would think that for quickpicks (which are about choosing an option) are more like a combo box than an input.

Reply:
maybe this should be set to input when it's an input box not a quick pick

Reply:
This element is the actual input box part of the quickpick I think? Nothing can be chosen from within it - a user navigates to the list of items to do so.

Reply:
I will defer to you as you own this area. If you think `combobox` is most appropriate, feel free to close the issue with that explanation.

Reply:
>  a user navigates to the list of items to do so.

The user can do it this way by using TAB, but the default experience is that the input has Dom focus and key handlers attached to the input allow for choosing and selecting of the item. A close comparison would be this combobox example:
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/